docs/docs/rasa-pro-setup.mdx

```diff
@@ -0,0 +1,120 @@
+---
+id: rasa-pro-setup
+sidebar_label: Rasa Pro Setup
+title: Rasa Pro Setup
+description: Installation of Rasa Pro
+abstract: Install Rasa Pro in your production environment to create an enterprise ready Rasa deployment.
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Rasa Pro is compromised of two packages. The packages need to be installed
+separately and in different environments:
+
+- **Rasa Plus**, a drop in replacement for Rasa Open Source for an
+  enterprise deployment. Rasa Plus should be used to deploy your assistant
+  into production.
+- **Rasa Pro Services**, flexible infrastructure and APIs on top of Rasa
+  Open Source. Rasa Pro Services should be deployed alongside, but separately
+  from your production assistant. Rasa Pro Services will connect to your
+  production assistant using Kafka.
```

Comment:
Does this mean a ccustomer using a different event broker can't use this service?

Reply:
Similar question to Anca's. Is this relevant to setting up Rasa Pro Services? We could include another feature tomorrow that has nothing to do with the event broker right?

Reply:
> Does this mean a ccustomer using a different event broker can't use this service?

Yes exactly. At this time we are only consuming from kafka. With the python solution we got a lot more flexibility there though and could provide this for other brokers.

Reply:
Very valid, @chandrikas I'll move this to the analytics section.